### ISSUE TITLE
Volume does not change

### ISSUE THREAD

**Issue by moale** [grzegorz914/homebridge-lgwebos-tv#228]
Hi, i get the below error while i attempt to change to volume

```
[LgWebOsTv] Device: 192.168.200.109 LG TV, set Volume error: ReferenceError: volume is not defined
[26/10/2024, 11:01:52] [LgWebOsTv] Device: 192.168.200.109 LG TV, set Volume error: ReferenceError: volume is not defined
[26/10/2024, 11:01:52] [LgWebOsTv] Device: 192.168.200.109 LG TV, set Volume error: ReferenceError: volume is not defined
[26/10/2024, 11:01:52] [LgWebOsTv] Device: 192.168.200.109 LG TV, set Volume error: ReferenceError: volume is not defined
[26/10/2024, 11:01:52] [LgWebOsTv] Device: 192.168.200.109 LG TV, set Volume error: ReferenceError: volume is not defined
```

If you set the slider to 0, TV goes in mute mode and you get

```
 Device: 192.168.200.109 LG TV, debug: Socket send: {"id":"f5368ce6000b","type":"request","uri":"ssap://audio/setMute","payload":{"mute":true}}
[26/10/2024, 11:10:18] [LgWebOsTv] Device: 192.168.200.109 LG TV, set Mute: ON
[26/10/2024, 11:10:18] [LgWebOsTv] Device: 192.168.200.109 LG TV, debug: Audio: {
  "changed": [
    "mute"
  ],
  "scenario": "mastervolume_tv_speaker",
  "active": false,
  "action": "changed",
  "volume": 31,
  "returnValue": true,
  "mute": true
}
[26/10/2024, 11:10:18] [LgWebOsTv] Device: 192.168.200.109 LG TV, Volume: 31%
[26/10/2024, 11:10:18] [LgWebOsTv] Device: 192.168.200.109 LG TV, Mute: ON
[26/10/2024, 11:10:18] [LgWebOsTv] Device: 192.168.200.109 LG TV, debug: Audio: {
  "returnValue": true
}
```

Thanks 
Alex

**Comment by moale** [comment]
awesome thanks!

**Comment by Kohle81** [comment]
Hi,
I have updated to 3.2.1, but Volume on LG TV doesn‘t changed. Loook at Plugin debug log.

[homebridge-lgwebos-tv.log.txt](https://github.com/user-attachments/files/17533288/homebridge-lgwebos-tv.log.txt)

**Comment by grzegorz914** [comment]
With hardware buttons RC or with slider? How do You change the volume?

**Comment by Kohle81** [comment]
With slider. I changed slider from 9 % to 20 %
![IMG_0844](https://github.com/user-attachments/assets/ca230386-6eb3-4992-88bc-33d36d4f6511)

**Comment by grzegorz914** [comment]
Already tested and slider and fan working correct

**Comment by Kohle81** [comment]
Thank you for this feedback. Then the error must be with my LG TV. If I change the status using the remote control, the volume is displayed correctly. Changing it using Homekit is not possible. I have now found another solution, so I don't want to be a nuisance any longer. Best regards and thank you for this plugin, Manuel

**Comment by grzegorz914** [comment]
In your debug I don’t see that You try to volume changed, can You try this and post the debug again.

**Comment by Kohle81** [comment]
Hello, thank you for your invitation to post a new log. I now wanted to increase the volume using the slider.
[homebridge-lgwebos-tv.log 2.txt](https://github.com/user-attachments/files/17577064/homebridge-lgwebos-tv.log.2.txt)

**Comment by Kohle81** [comment]
It works!!!!!Thanks a lot for fixing!Am 30.10.2024 um 19:57 schrieb Grzegorz ***@***.***>:﻿
@Kohle81 fixed in 3.2.4
